packages/runner/src/suite.ts

```diff
@@ -60,7 +60,7 @@ function createSuiteCollector(name: string, factory: SuiteFactory = () => { }, m
   initSuite()
 
   const test = createTest(function (name: string, fn = noop, options = suiteOptions) {
-    const mode = this.only ? 'only' : this.skip ? 'skip' : this.todo ? 'todo' : 'run'
+    const mode = this.only ? 'only' : this.skip ? 'skip' : this.todo ? 'todo' : this.repeats ? 'repeats' : 'run'
```

Comment:
We don't want `repeats` mode

Reply:
Oh, strange that this didn't show an error, will remove it